src/Currencies.java

```diff
@@ -0,0 +1,22 @@
+import java.util.*;
+public class Currencies {
+    Map<String, Map<String,Double>> rates;
+    Currencies(){
+        rates= new HashMap<>();
```

Comment:
actually this initialisation of variable can be done in the palce where it is being declared, you do not have to put it to constructor, you can remove the constructor and use the default one.